TestNull

```diff
@@ -0,0 +1,30 @@
+import java.util.LinkedHashSet;
+ 
+public class SimpleLinkedHashSetExample {
+ 
+  public static void main(String[] args) {
+    //create object of LinkedHashSet
+    LinkedHashSet lhashSet = new LinkedHashSet();
+   
+    /*
+      Add an Object to LinkedHashSet using
+      boolean add(Object obj) method of Java LinkedHashSet class.
+      This method adds an element to LinkedHashSet if it is not
+      already present in LinkedHashSet.
```

Comment:
import java.util.LinkedHashSet;
 
public class SimpleLinkedHashSetExample {
 
  public static void main(String[] args) {
    //create object of LinkedHashSet
    LinkedHashSet lhashSet = new LinkedHashSet();